src/app/components/buscadorTurnosPrestaciones/turnos-prestaciones.html

```diff
@@ -126,7 +126,7 @@
                 </div>
                 <div class="col d-flex justify-content-end">
                     <plex-button label="Cerrar" type="danger" (click)="onClose()"></plex-button>
-                    <plex-button *ngIf="(prestacion.financiador?.financiador === 'SUMAR' || prestacion.financiador === 'SUMAR') && ((!prestacion.estadoFacturacion) || prestacion.estadoFacturacion?.estado!=='Comprobante con prestacion')"
+                    <plex-button *ngIf="(this.auth.profesional===null) && (prestacion.financiador?.financiador === 'SUMAR' || prestacion.financiador === 'SUMAR') && ((!prestacion.estadoFacturacion) || prestacion.estadoFacturacion?.estado!=='Comprobante con prestacion')"
```

Comment:
Sólo en el caso del this.auth.profesional === null?, probar de reemplazarlo por *ngIf="(!this.auth.profesional) && .....

Reply:
Testeado y modificado